src/components/views/rooms/RoomPreviewBar.tsx

```diff
@@ -397,6 +402,21 @@ export default class RoomPreviewBar extends React.Component<IProps, IState> {
                 }
                 break;
             }
+            case MessageCase.RequestDenied: {
+                title = _t("You have been denied access");
+
+                subTitle = _t(
+                    "As you have been denied access, please wait for an invitation from a room admin or moderator.",
```

Comment:
The [designs on the linked ticket](https://github.com/vector-im/element-web/issues/18655#issuecomment-1570432810) use different copy - is there a reason this is different?

Reply:
you are right! the ticket on our side had different text! but i added the text from the design!